apina-core/src/main/kotlin/fi/evident/apina/model/Endpoint.kt

```diff
@@ -19,7 +20,8 @@ class Endpoint(
     /** URI template for the endpoint  */
     val uriTemplate: URITemplate,
     val responseBody: ApiType?,
-    val generateUrlMethod: Boolean
+    val generateUrlMethod: Boolean,
+    val optionalTypeMode: OptionalTypeMode
```

Comment:
`optionalTypeMode` is surely not a property of an `Endpoint`, but of the translation process. The `toString` is just debugging output and the fact that it uses TypeScript-representation is just accidental. So you can pick arbitrary type mode for that, probably just NULL to keep the format same as previously.

Reply:
Dropped this one and used NULL in toString().